server/app.module.ts

```diff
@@ -112,6 +113,9 @@ export class AppModule implements NestModule {
                 })
             );
         }
+        if (options.config.novu) {
```

Comment:
chore: let's create a task to re-evaluate this approach. If the module is not registered I believe the services depending on it will also crash, we need something more decoupled, like a message system or something like that.